docs/pages/use-teleport/teleport-connect.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Using Teleport Connect
+description: Using Teleport Connect
+---
+
+Teleport Connect provides easy and secure access to SSH servers and databases, with support for
+other resources such as Kubernetes clusters and application access coming in the future.
+
+![resources tab in Teleport Connect](../../img/use-teleport/connect-cluster.png)
+
+## Installation & Upgrade
+
+Head over to the [Downloads](https://goteleport.com/download/) page to download the most recent
+version of Teleport Connect. As of now, Teleport Connect supports macOS only. Double-click the
+downloaded .dmg file and drag the Teleport Connect icon to the Applications folder.
+
+To upgrade Teleport Connect to a newer version, download the .dmg file with that version and follow
+the installation steps.
+
+## Interface Overview
+
+![interface overview of Teleport Connect](../../img/use-teleport/connect-ui-overview.png)
+
+The top bar of Teleport Connect consists of:
+
+- The profile selector (the top right) allowing you to switch between profiles on different Teleport
+  clusters as well as log in or out of the clusters.
+- The connection list (the top left) showing recent connections, allowing you to seamlessly switch
+  between them.
+- The command bar (in the middle) which launches any command you input in a new terminal tab.
+- The cluster selector (to the left of the command bar) shows up only if there are leaf clusters
+  connected to the root cluster. It lets you browse leaf cluster resources. It also changes
+  which cluster the commands in the command bar are going to target.
+
+## Connecting to an SSH Server
+
+1. Open a tab with cluster resources by clicking on the plus symbol at the end of the tab bar. You
+   can also press `Cmd + T` to achieve the same result.
+2. Look for the SSH server you want to connect to and click the Connect button to the right.
+3. Select or enter the login you wish to use and press `Enter`.
+4. A new tab will open with a shell session on the chosen server.
+
+Alternatively, you can type `tsh ssh` into the command bar and the autocompletion will guide you to
+connect to a server.
```

Comment:
will guide you "through the server connection process" or "will guide you through connecting to a server" or "will help you connect to a server"